### DISCUSSION TITLE
Is there any output plugin to remove all 404 lines in log files, before forwarding to aggregator

### DISCUSSION THREAD

**Discussion by sirdatta** [fluent/fluent-bit#8602] [Q&A]
I would like to a map reduce on certain HTTP error codes and HTTP verbs. Are there any plugins that accomplish this?

**Comment by patrick-stephens** [comment]
Sure, you can parse them into key-values then use grep to drop them based on the key-value. Or even use grep on the raw log lines to drop if it sees a particular regex but I'd suggest parsing first, e.g. with the default nginx parser into the actual response code and then just ensure you drop 404 values there rather than a 404 anywhere in the log line.
